packages/TCSLib/TCSLib.0.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: ["Oliver Friedmann" "Martin Lange"]
+homepage: "https://github.com/tcsprojects/tcslib"
+bug-reports: "https://github.com/tcsprojects/tcslib/issues"
+license: "BSD-3-clause"
+dev-repo: "git://github.com/tcsprojects/tcslib.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: ["ocamlfind" "remove" "Camldiets"]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+]
```

Comment:
This is [failing](https://travis-ci.org/ocaml/opam-repository/jobs/206978505) because it uses `String.lowercase_ascii`, which was [added in OCaml 4.03.0](https://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html#VALlowercase_ascii)

```
File "src/utils/tcsargs.ml", line 162, characters 46-68:
# Error: Unbound value String.lowercase_ascii
# Command exited with code 2.
```
@oliverfriedmann: could you please add a version constraint to the `opam` file?

```
available: [ ocaml-version >= "4.03.0" ]
```